NAS-100260 Updated intro to include the new quota exceeded on dataset option. (#833)
freenas/freenas-docs: userguide/storage.rst

.. table:: ZFS Dataset Options
   :class: longtable

   +-----------------------+-----------------+----------------+--------------------------------------------------------------------------------------------+
   | Setting               | Value           | Advanced Mode  | Description                                                                                |
   |                       |                 |                |                                                                                            |
   +=======================+=================+================+============================================================================================+

>    | Dataset Name          | string          |                | Enter a mandatory unique name for the dataset.                                             |

Reviewer: Enter a unique name for the dataset. Required.

   |                       |                 |                |                                                                                            |
   +-----------------------+-----------------+----------------+--------------------------------------------------------------------------------------------+
   | Comments              | string          |                | Enter optional comments or notes about this dataset.                                       |
   |                       |                 |                |                                                                                            |
   +-----------------------+-----------------+----------------+--------------------------------------------------------------------------------------------+

>    | Sync                  | drop-down       |                | Sets the data write synchronization.                                                       |

Reviewer: s/Sets/Set/

   |                       |                 | prevent users from taking all available space.                                                              |
   |                       |                 |                                                                                                             |
   +-----------------------+-----------------+-------------------------------------------------------------------------------------------------------------+

>    | Quota warning         | integer         | Only available in :guilabel:`Advanced Mode`.                                                                |

Reviewer: Any table with Advanced Mode should use checkmarks in a column.

>    | alert at, %           |                 | Issues a warning alert when the dataset quota has reached the specifed value in percent.                    |

Reviewer: ```suggestion
   | alert at, %           |                 | Show an alert when the dataset quota reaches the specified value in percent. 
```

>    |                       |                 | To inherit the dataset values of the parent, leave blank. To disable this setting, enter *0*.               |

Reviewer: ```suggestion
   |                       |                 | Leave blank to inherit parent dataset value, or enter *0* to disable.
```

   |                       |                 |                                                                                                             |
   +-----------------------+-----------------+-------------------------------------------------------------------------------------------------------------+

>    | Quota critical        | integer         | Only available in :guilabel:`Advanced Mode`.                                                                |

Reviewer: Checkbox for column.

>    | alert at, %           |                 | Issues a critical alert when the dataset quota has reached the specified value in percent.                  |

Reviewer: As above.

>    |                       |                 | To inherit the dataset values of the parent, leave blank. TO disable this setting, enter *0*.               |

Reviewer: As above.

   |                       |                 |                                                                                                             |
   +-----------------------+-----------------+-------------------------------------------------------------------------------------------------------------+
   | Quota for this        | integer         | Only available in :guilabel:`Advanced Mode`.                                                                |
   | dataset and all       |                 | A specified value applies to both this dataset and any child datasets.                                      |
   | children              |                 |                                                                                                             |
   |                       |                 |                                                                                                             |
   +-----------------------+-----------------+-------------------------------------------------------------------------------------------------------------+

>    | Quota warning         | integer         | Only available in :guilabel:`Advanced Mode`.                                                                |

Reviewer: Is this a duplicate of the one above?

Author: No. In the old UI they appear how they are in the table.

   | alert at, %           |                 | Issues a warning alert when the dataset quota has reached the specifed value in percent.                    |
   |                       |                 | To inherit the dataset values of the parent, leave blank. To disable this setting, enter *0*.               |
   |                       |                 |                                                                                                             |
   +-----------------------+-----------------+-------------------------------------------------------------------------------------------------------------+

>    | Quota critical        | integer         | Only available in :guilabel:`Advanced Mode`.                                                                |

Reviewer: Too many pastes?

Author: No. In the old UI they appear how they are in the table.
